ERA5/construct_era5_daily_cancities_revised.py

```diff
@@ -0,0 +1,444 @@
+"""
+Simple script to construct xclim's ERA5/daily_surface_cancities_1990-1993.nc test dataset.
+
+This dataset has the goal to provide a real example of every variable supported by xclim
+in the atmos realm. When a variable is not given by ERA5, either an approximated version
+is computed or it is ensured that needed base variables are given here. For example, the
+`snw` variable doesn't need to be in the dataset, it is simply computed with swe * 1000.
+
+Some of those extra variables are computed here but not included in the final output.
+
+Author: Pascal Bourgault, 2021
+Revised: Trevor James Smith, 2022
+"""
+import datetime as dt
+import logging
+import sys
+from pathlib import Path
+
+import xarray as xr
+import xclim as xc
+from dask import compute
+from dask.distributed import Client
+from xclim.core import formatting
+from xclim.core.units import convert_units_to
+
+logging.basicConfig(level=logging.INFO)
+
+# if len(sys.argv) == 2:
+#     base_path = Path(sys.argv[-1])
+# else:
+#     base_path = Path().cwd()
+
+base_path = Path("/tank/scenario")
```

Comment:
```suggestion
if len(sys.argv) == 2:
    base_path = Path(sys.argv[-1])
else:
    base_path = Path().cwd()
```
Achtung!

Reply:
I'll be performing a squash merge to remove this. My bad!